web_drop_target/static/src/js/web_drop_target.js

```diff
@@ -6,14 +6,14 @@
 odoo.define('web_drop_target', function(require) {
     var Model = require('web.Model'),
         FormView = require('web.FormView'),
-        core = require('web.core');
-    var _t = core._t;
+        core = require('web.core'),
+        qweb = core.qweb;
```

Comment:
sorry for this one, the linter disagrees with me: https://eslint.org/docs/rules/no-mixed-requires (TIL)

Reply:
Done